platform-sdk/swirlds-config-impl/src/main/java/module-info.java

```diff
@@ -7,8 +7,11 @@
     exports com.swirlds.config.impl.validators;
 
     requires transitive com.swirlds.config.api;
+    requires com.hedera.node.config;
     requires com.swirlds.base;
     requires com.swirlds.config.extensions;
+    requires com.fasterxml.jackson.core;
+    requires com.fasterxml.jackson.databind;
```

Comment:
Out of curiosity, why do we need these dependencies in the platform code?

Reply:
Because of the changes in `ConfigDataFactory` to be able to use `ObjectMapper` and parse `CloudBucketConfig` objects

Reply:
Sounds good. Thanks!